workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -297,7 +297,10 @@ def _get_postphysics_stepper(
             self._log_info("Using MLStepper for postphysics updates")
             model = self._open_model(config.scikit_learn, "_postphysics")
             stepper: Optional[Stepper] = PureMLStepper(
-                model, self._timestep, hydrostatic
+                model,
+                self._timestep,
+                hydrostatic,
+                config.scikit_learn.use_mse_conserving_humidity_limiter,
```

Comment:
As an extra precaution, can you log which specific humidity limiter is used at initialization?

Reply:
It will be recorded in the `fv3config.yml`, but yeah we can also log.